js/form_editor_controller.js

```diff
@@ -136,7 +136,10 @@ class GlpiFormEditorController
         $(document)
             .on(
                 'click',
-                () => {
+                event => {
+                    if (event.isDefaultPrevented)
+                        return;
+
```

Comment:
I suppose this was needed because clicking on an option would trigger the set active function ?
Have you tried calling `event.stopPropagation()` in your handler ?